### PR TITLE
License under MIT

### PR DESCRIPTION
**Pull request by geophree**
Noticed your code was unlicensed, and I was interested in maybe using it. This pull request licenses it under MIT, I used the name on your github profile.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Clament John
```
Comment (clmnin) [inline LICENSE]: This was originally a youtube video by @michaeljones. So please him too.
```suggestion
Copyright (c) 2021 Michael Jones and Clament John
```

**Review by clmnin — CHANGES_REQUESTED** [review]
Please add Michael Jones as one of the owners to this code. Other than that LGTM

**Comment by geophree** [comment]
Updated.

**Comment by clmnin** [comment]
Thank you

**Pull request merged**